gunicorn/workers/ggevent.py

```diff
@@ -51,22 +50,15 @@ class GeventWorker(AsyncWorker):
     wsgi_handler = None
 
     def patch(self):
-        from gevent import monkey
-        monkey.noisy = False
-
-        # if the new version is used make sure to patch subprocess
-        if gevent.version_info[0] == 0:
-            monkey.patch_all()
-        else:
-            monkey.patch_all(subprocess=True)
+        monkey.patch_all()
 
         # monkey patch sendfile to make it none blocking
         patch_sendfile()
 
         # patch sockets
         sockets = []
         for s in self.sockets:
-            sockets.append(socket(s.FAMILY, _socket.SOCK_STREAM,
+            sockets.append(socket(s.FAMILY, socket.SOCK_STREAM,
```

Comment:
Pretty sure this will fail with `AttributeError`, because above there is `from gevent.socket import socket`, so `socket` refers to a class, not a module. Either `from gevent import socket` and then `socket.socket` as needed, or `from gevent.socket import SOCK_STREAM`?